addons/patches/scifisupportplus/XEH_postInit_list.sqf

```diff
@@ -0,0 +1,50 @@
+#include "script_component.hpp"
+
+/* 
+ * Append SWS ground vehicles to the following variables:
+ * PHAN_UNSC_GV_list
+ * PHAN_UNSC_GV_listDISPLAY
+ *
+ * Follows pattern defined in PHAN_ScifiSupportPlus/bootstrap/xeh_postinit_list.sqf
+ */
+
+call compile preprocessFileLineNumbers 'PHAN_ScifiSupportPlus\bootstrap\xeh_postinit_list.sqf';
+
+private _sws_gv_list_insert = [
+	"SWS_V_APC_M107_Beaver",
+	"SWS_V_APC_M35_Cougar",
+	"SWS_V_MBT_M700_Viper"
+];
+
+{
+	if (_x!="" && (gettext (configFile >> "Cfgvehicles" >> _x >> "displayname")) != "") then 
+	{
+		private _color = switch ((getNumber (configFile >> "Cfgvehicles" >> _x >> "side"))) do {
```

Comment:
Something to note is that switch/case has a much worse performance in SQF than select, so with something like this you could do
```sqf
  private _cfg = _x;
  private _side = [_cfg, "side", 4 /* sideUnknown */] call BIS_fnc_returnConfigEntry;
  private _color /* [R,G,B,A] */ = [
    [255, 0, 0, 255],
    [0, 0, 255, 255],
    [0, 255, 0, 255],
    [255, 0, 255, 255],
    [0, 0, 0, 255]
  ] select (_side min 4);
```
Not entirely necessary in this case, but it's worth noting for things that might happen a lot more frequently.